ui/app/serializers/clients/new-init-activity.js

```diff
@@ -1,3 +1,3 @@
 import ApplicationSerializer from '../application';
 
-export default ApplicationSerializer.extend({});
+export default class newInitActivity extends ApplicationSerializer {}
```

Comment:
Smallest of nits but class name could be capitalized and it might be a good idea to add serializer to the name as in `NewInitActivitySerializer` to differentiate it from the model class name.

Reply:
good NIT, just missed it. Thanks!